### PR TITLE
recreating CNV-17623: RN for NMO

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.11 RN

Issue: [CNV-17623](https://issues.redhat.com//browse/CNV-17623)

Link to docs preview: http://file.rdu.redhat.com/sjess/CNV17623B2/virt/virt-4-11-release-notes.html

Note: Original PR with reviewer edits (https://github.com/openshift/openshift-docs/pull/48422) closed because of squash issues.

### CODE REVIEW

**Comment by dbasunag** [comment]
lgtm

File: virt/virt-4-11-release-notes.adoc [hunk 1]
```diff
@@ -130,6 +128,10 @@ To preserve and support your existing nmstate configuration, install the xref:..
 
 //CNV-18323 VM templates can be made public i.e. available to all users
 
+* As of version 4.11, `node-maintenance-operator` (NMO) is no longer shipped with {VirtProductName}. It is now available to deploy as a standalone operator through the Operator Lifecycle Manager (OLM) in the OperatorHub.
```
Comment (sjhala-ccs) [inline virt/virt-4-11-release-notes.adoc:131]: Suggestion to rephrase the second sentence as something like this:
````
You can now deploy it as a standalone Operator from the *OperatorHub* in the {product-title} web console, or by using the OpenShift CLI (`oc`).
````

File: virt/virt-4-11-release-notes.adoc [hunk 2]
```diff
@@ -130,6 +128,10 @@ To preserve and support your existing nmstate configuration, install the xref:..
...
++
+If you are using {VirtProductName} 4.10.2 or later, you can upgrade to {product-title} 4.11, but will be blocked from upgrading {VirtProductName} if the non-standalone NMO is still in use. To upgrade to {VirtProductName} 4.11, you must move all nodes out of maintenance mode, or the cluster admin must install the standalone NMO and replace the `nodemaintenances.nodemaintenance.kubevirt.io` CR with a `nodemaintenances.nodemaintenance.medik8s.io` CR.
```
Comment (sjhala-ccs) [inline virt/virt-4-11-release-notes.adoc:133]: I think this paragraph could be made more concise. Suggestion:
````
You must perform one of the following tasks before updating to {VirtProductName} {VirtVersion} from {VirtProductName} 4.10.2 and later releases:

- Move all nodes out of maintenance mode
- Install the standalone Node Maintenance Operator and replace the `nodemaintenances.nodemaintenance.kubevirt.io` custom resource (CR) with a `nodemaintenances.nodemaintenance.medik8s.io` CR
````

**Review by sjhala-ccs — COMMENTED** [review]
LGTM

**Pull request merged**